index.html

```diff
@@ -14,7 +15,7 @@
       <h2>The Main Content</h2>
       <div class='left'>
```

Comment:
Missing semantic tag - this div could be a `nav`, etc, the div with a class of main could be a `main`